geomesa-hbase/pom.xml

```diff
@@ -32,6 +32,11 @@
                 <artifactId>geomesa-hbase-datastore_${scala.binary.version}</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.google.guava</groupId>
+                <artifactId>guava</artifactId>
+                <version>${hbase.guava.version}</version>
```

Comment:
is this the same for bigtable? modifying it here affects those modules as well - i think is why it was not done before